### ISSUE TITLE
[V2][Android] Bottombar Image Loader causes a crash

### ISSUE THREAD

**Issue by andresesfm** [wix/react-native-navigation#3131]
### Issue Description

Running stress tests (Monkey runner) on the app, intermittently crashes with the following stacktrace:
### Steps to Reproduce / Code Snippets / Screenshots
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.setTag(java.lang.Object)' on a null object reference
        at com.reactnativenavigation.views.BottomTabs.setTabTag(BottomTabs.java:21)
        at com.reactnativenavigation.viewcontrollers.bottomtabs.BottomTabsController$1.lambda$onComplete$0$BottomTabsController$1(BottomTabsController.java:122)
        at com.reactnativenavigation.viewcontrollers.bottomtabs.BottomTabsController$1$$Lambda$0.run(Unknown Source)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

```

### Proposed Fix

https://github.com/wix/react-native-navigation/pull/3132
---
### Environment
* React Native Navigation version: V2
* React Native version:  0.53.3
* Platform(s) (iOS, Android, or both?): Android
* Device info (Simulator/Device? OS version? Debug/Release?): Emulator / Debug

**Comment by guyca** [comment]
Interesting, I haven't encountered this crash when running our Detox e2e suite.
I'm wondering if using `addItems(List<AHBottomNavigationItem> items)` instead of `addItem(AHBottomNavigationItem item)` will help.

**Comment by andresesfm** [comment]
Hello @guyca 
TL;DR: Even using `addItems` we'd need this change due to the async nature of `post`

Yes, it would certainly help to add all items at once. However, since we this operation is async `post( ()->...`: https://github.com/wix/react-native-navigation/blob/v2/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/bottomtabs/BottomTabsController.java#L123 , this crash would still happen albeit not as often.

The root problem is comes from `AHBottomNavigation` implementation of `adItem` calling `createItems`: https://github.com/aurelhubert/ahbottomnavigation/blob/master/ahbottomnavigation/src/main/java/com/aurelhubert/ahbottomnavigation/AHBottomNavigation.java#L975 that discards the LinearLayout and builds it from scratch again: https://github.com/aurelhubert/ahbottomnavigation/blob/master/ahbottomnavigation/src/main/java/com/aurelhubert/ahbottomnavigation/AHBottomNavigation.java#L274

I will create a separate ticket so we can start the discussion on the `addItems` implementation

**Comment by andresesfm** [comment]
Here is the issue I created for using `addItems`: https://github.com/wix/react-native-navigation/issues/3140

**Comment by guyca** [comment]
Seems like If we add all items at once, after images are resolved we could set testId synchronously and avoid calling post.

**Comment by guyca** [comment]
merged and resolved 👍
